src/Uno.UI/UI/Xaml/Controls/SwipeControl/SwipeItem.cs

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// IconSource is implemented in WUX in the OS repo, so we don't need to
+// include IconSource.h on that side.
+
+// Imported in uno on 2021/03/21 from commit 307bd99682cccaa128483036b764c0b7c862d666
+// https://github.com/microsoft/microsoft-ui-xaml/blob/307bd99682cccaa128483036b764c0b7c862d666/dev/SwipeControl/SwipeItem.cpp
+
+using System;
+using System.Windows.Input;
+using Windows.Devices.Input;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class SwipeItem : FrameworkElement
+	{
+		//static double s_swipeItemWidth = 68.0;
```

Comment:
Was it unused in the original source ?

Reply:
I search in the whole original source repo, apparently it wasn't used.